ui/src/pages/Dashboard/DataIngestionModal.tsx

```diff
@@ -0,0 +1,72 @@
+import Modal from '@splunk/react-ui/Modal';
+import React, { ReactElement } from 'react';
+import styled from 'styled-components';
+
+import { variables } from '@splunk/themes';
+import Button from '@splunk/react-ui/Button';
+import { StyledButton } from '../EntryPageStyle';
+
+const ModalWrapper = styled(Modal)`
+    width: 60%;
+    height: 80%;
+    margin-top: 4vh;
+`;
+
+const ModalHeader = styled(Modal.Header)`
+    background-color: ${variables.backgroundColorHover};
+`;
+const ModalFooter = styled(Modal.Footer)`
+    background-color: ${variables.backgroundColorHover};
+`;
+const ModalBody = styled(Modal.Body)`
+    background-color: ${variables.backgroundColorHover};
+`;
+
+const StyledDiv = styled('div')`
+    display: grid;
+    grid-template-columns: 0.35fr 1fr;
+    margin: 0px 10px;
+
+    .footerBtn:first-child {
+        justify-self: start;
```

Comment:
can it be justify content space between for the parent?

Reply:
I have tried that but it's taking the full width for both the buttons

const StyledDiv = styled('div')`
    display: flex;
    justify-content: space-between;
    margin: 0px 10px;

    .footerBtn {
        margin: 0 auto;
    }
`;